app/src/components/evals/EvalContentTabs/Settings/Settings.tsx

```diff
@@ -117,39 +119,26 @@ const Settings = () => {
     includedModelIds,
     datasetEval?.instructions,
     instructions,
+    datasetEval?.evaluationModelId,
+    selectedEvaluationModel.id,
   ]);
 
-  const modelOptions = useMemo(() => {
-    const options = [
-      {
-        id: ORIGINAL_MODEL_ID,
-        name: getOutputTitle(ORIGINAL_MODEL_ID) ?? "",
-      },
-    ];
-    if (!dataset) return options;
-    return [
-      ...options,
-      ...dataset.enabledComparisonModels.map((comparisonModelId) => ({
-        id: comparisonModelId,
-        name: getOutputTitle(comparisonModelId) ?? "",
-      })),
-      ...dataset.deployedFineTunes.map((model) => ({
-        id: model.id,
-        name: getOutputTitle(model.id, model.slug) ?? "",
-      })),
-    ];
-  }, [dataset]);
-
   const reset = useCallback(() => {
     if (datasetEval) {
       setName(datasetEval.name);
       setInstructions(datasetEval.instructions ?? "");
       setNumRows(datasetEval.numRows);
       setIncludedModelIds(datasetEval.outputSources.map((source) => source.modelId));
+
+      usedEvaluationModel && setSelectedEvaluationModel(usedEvaluationModel);
     }
   }, [datasetEval, setName, setInstructions, setNumRows, setIncludedModelIds]);
 
-  useEffect(() => reset(), [datasetEval, reset]);
+  useEffect(() => {
+    usedEvaluationModel && setSelectedEvaluationModel(usedEvaluationModel);
+  }, [usedEvaluationModel]);
+
+  useEffect(() => void reset(), [datasetEval, reset]);
```

Comment:
Is reset async? I don’t think you need to use void when calling it.

Reply:
Oh yes, I forgot to remove it, thank you!